src/main/java/io/netty/incubator/codec/quic/QuicheQuicStreamChannel.java

```diff
@@ -396,7 +396,7 @@ boolean writable(@SuppressWarnings("unused") int capacity) {
         assert eventLoop().inEventLoop();
         this.capacity = capacity;
         boolean mayNeedWrite = ((QuicStreamChannelUnsafe) unsafe()).writeQueued();
-        updateWritabilityIfNeeded(capacity > 0);
+        updateWritabilityIfNeeded(this.capacity > 0);
```

Comment:
nit: It isn't immediately obvious why we need to re-read the field as it is always updated on the event loop. Suggest; adding a comment that `writeQueued()` may update capacity and hence we need to re-read.

Reply:
I added one..